core/transport/src/main/proto/proto.lock

```diff
@@ -2751,6 +2751,11 @@
                 "id": 6,
                 "name": "disableAreDescendantsLoadedCheck",
                 "type": "bool"
+              },
```

Comment:
this change should've been included as part of https://github.com/Alluxio/alluxio/pull/16893 @flaming-archer 
the proto.lock file needs to be updated whenever proto files are changed to ensure backcompat